modules/league.py

```diff
@@ -16,6 +18,8 @@
 
 logger = logging.getLogger('polybot.' + __name__)
 
+STANDINGS = defaultdict(lambda: False)
```

Comment:
`STANDINGS` is global but not constant. According to PEP 8, uppercase is only for constants.